### PR TITLE
MrEnder website

### PR DESCRIPTION
**Pull request by MrEnder0**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
Website can't be a coming soon page.

File: domains/mrender.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "redirects to website",
+    "repo": "https://github.com/MrEnder0/MrEnder0.github.io",
+    "owner": {
+        "username": "MrEnder0",
+        "email": ""
+        "discord": "MrEnder#0001"
+    },
+    "record": {
+        "CNAME": "MrEnder0.github.io"
```
Comment (TheBotlyNoob) [inline domains/mrender.json:10]: As said above, the website cannot be a "coming soon" page. If you want to can change it to 
```suggestion
        "URL": "https://github.com/MrEnder0"
```
Which will redirect to your GitHub profile.

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]

**Review by wdhdev — APPROVED** [review]

**Comment by MrEnder0** [comment]
Hey I don't have much experience with pull requests. Am I supposed to be doing something or just leave the rest up to you guys?

**Comment by WilliamDavidHarrison** [comment]
> Hey I don't have much experience with pull requests. Am I supposed to be doing something or just leave the rest up to you guys?

Leave the rest up to us 😃

**Review by MrEnder0 — COMMENTED** [review]
re-submit

**Pull request closed without merging**